so_pysm_models/get_so_model.py

```diff
@@ -0,0 +1,102 @@
+import healpy as hp
+import numpy as np
+import pysm
+import os
+from astropy.utils import data
+from pysm.common import read_map, loadtxt
+
+data_dir = os.path.join(os.path.dirname(__file__), 'so_template')
+template = lambda x: os.path.join(data_dir, x)
+DATAURL = "http://portal.nersc.gov/project/cmb/so_pysm_models_data/"
+
+def get_data_from_url(filename):
+    with data.conf.set_temp("dataurl", DATAURL), data.conf.set_temp(
+        "remote_timeout", 30):
+        map_out = data.get_pkg_data_filename(filename)
+    return map_out
+
+def models(key, nside, pixel_indices=None, mpi_comm=None, small_scale=False):
```

Comment:
I wonder if you can avoid this function altogether and directly use `SO_d0`

Reply:
also because otherwise how can you pass the `smallscale` keyword argument?

Reply:
I don't understand this.. Which function are you referring to? 
right now to produce a high res maps I'm doing something like:
`dust_model = "SO_d0"`
`sky = pysm.Sky({"dust":models(dust_model, nside, small_scale=True)})`
`dust_map = sky.dust(353.)`

Reply:
let's call it `so_models` instead of `models` so we can mix it more easily with the one from PySM

Reply:
done